### PR TITLE
feat: Alerts component

### PR DESCRIPTION
**Pull request by Golodhros**
### Summary of Changes
Adds the alerts component
Includes tests and a SVG icon for the alert

### Documentation
Added Storybook story for the component

Components___Alert_-_Alert_⋅_Storybook

Attributes___Icons_-_SVG_Icons_⋅_Storybook

### CheckList

Make sure you have checked **all** steps below to ensure a timely review.

- [ ] PR title addresses the issue accurately and concisely. Example: "Updates the version of Flask to v1.0.2"
  - In case you are adding a dependency, check if the license complies with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
- [ ] PR includes a summary of changes, including screenshots of any UI changes.
- [ ] PR adds unit tests, updates existing unit tests, **OR** documents why no test additions or modifications are needed.
- [ ] In case of new functionality, my PR adds documentation that describes how to use it.
  - All the public python functions and the classes in the PR contain docstrings that explain what it does
- [ ] PR passes all tests documented in the [developer guide](https://github.com/lyft/amundsenfrontendlibrary/blob/master/docs/developer_guide.md#testing)

### CODE REVIEW

File: amundsen_application/static/js/components/common/Alert/AlertIcon.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+
+const STROKE_COLOR = '#b8072c'; // $red70
+const SIZE = 16;
+
+const AlertIcon: React.FC = () => {
+  return (
+    <svg
```
Comment (ttannis) [inline amundsen_application/static/js/components/common/Alert/AlertIcon.tsx:8]: Would it make sense to have a `common/icons` where we start putting new amundsen icons that use inline-svgs?

File: amundsen_application/static/js/components/common/Alert/index.tsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  message,
+  onAction,
+  actionText,
+}: AlertProps) => {
+  return (
+    <div className="alert">
+      <AlertIcon />
+      <p className="alert-message">{message}</p>
+      {actionText && onAction && (
+        <button type="button" className="btn btn-link" onClick={onAction}>
```
Comment (ttannis) [inline amundsen_application/static/js/components/common/Alert/index.tsx:26]: For our internal use case, the text will end up being a link to another Amundsen page. Given our desire to use the correct components for specific actions, do you think we should still support a button with onClick vs and anchor with an href...or both?

File: amundsen_application/static/js/components/common/Alert/AlertIcon.tsx (hunk 1 above)
Comment (Golodhros) [reply amundsen_application/static/js/components/common/Alert/AlertIcon.tsx:8]: Makes sense! I will move it there!

File: amundsen_application/static/js/components/common/Alert/index.tsx (hunk 2 above)
Comment (Golodhros) [reply amundsen_application/static/js/components/common/Alert/index.tsx:26]: Oh, I see.
I could make it support both easily. I will add an optional actionHref prop, if that has a value, I will render a router Link instead

**Pull request merged**